src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work!");
```

Comment:
The message should indicate what type of `Machine` is working. Consider rephrasing it to include the class name, for example: 'Excavator is working!'.